Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.yml

```
@@ -8,8 +8,8 @@ configuration:
   name: url
   required: true
   type: 0
-- display: API Key
-  additionalinfo: The Expanse API Key to use to connect.
+- additionalinfo: The Expanse API Key to use to connect.
```

```suggestion
- additionalinfo: Xpanse API Key to use to connect.
```
please change all `Expanse` appearances in the yml/readme to `Xpanse` (whenever possible, e.g. `display`, `description`).